addons/website_sale/static/src/xml/website_sale.editor.xml

```diff
@@ -60,4 +60,12 @@
     </div>
 </div>
 
+<div t-name="website_sale.dialog_confirmation">
+    Are you sure you want to modify the shop grid? After confirming you can not discard the changes.
+    <div class="form-group form-check mt-2 mb-0">
```

Comment:
Avoid text node next to tag node when possible. Put the text inside a `<p>` then you will be able to remove the `mt-2` class here as `<p>` has a natural bottom margin.